Tools/Schema.NET.Tool/Services/SchemaService.cs

```diff
@@ -220,7 +220,7 @@ private static void SetParentAndChildren(List<GeneratorSchemaClass> classes)
             HashSet<Uri> knownSchemaClasses,
             bool includePending)
         {
-            var className = StartsWithNumber.IsMatch(schemaClass.Label) ? $"_{schemaClass.Label}" : schemaClass.Label;
+            var className = StartsWithNumber.IsMatch(schemaClass.Label) ? $"Type{schemaClass.Label}" : schemaClass.Label;
```

Comment:
I guess we just need to mention this in the release notes?

Reply:
Only one pending type is affected.

Reply:
Ahhhh, OK - not a big deal then!